### PR TITLE
Scene editor - add ''Edit Object" command to r.click menu and a 'Delete' command

### PR DESCRIPTION
**Pull request by blurymind**
This will make it easier to edit a selected object in the instance editor, without having to find it in a long list of other objects

It also adds a 'Delete' command - for those who might not know the keyboard button for it

### CODE REVIEW

File: newIDE/app/src/SceneEditor/index.js [hunk 1]
```diff
@@ -205,6 +205,12 @@ export default class SceneEditor extends Component {
     this.setState({ scenePropertiesDialogOpen: open });
   };
 
+  openObjectEditor = () => {
+    if(!this.instancesSelection.getSelectedInstances()[0]) {return};
```
Comment (4ian) [inline newIDE/app/src/SceneEditor/index.js]: nitpicking: You can use `if (!this.instanceSelection.hasSelectedInstances())`.

File: newIDE/app/src/SceneEditor/index.js [hunk 2]
```diff
@@ -205,6 +205,12 @@ export default class SceneEditor extends Component {
...
+    const selectedObjectInstanceName = this.instancesSelection.getSelectedInstances()[0].getObjectName();
```
Comment (4ian) [inline newIDE/app/src/SceneEditor/index.js]: nitpicking: this variable represents the "name" of the "object" associated to the "selected instance". I would then name the variable `selectedInstanceObjectName`.

**Comment by 4ian** [comment]
Looks sensible! Check my two comments and don't forget to run prettier on the modified files. Then I think it's safe to merge. :)

File: newIDE/app/src/SceneEditor/index.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/SceneEditor/index.js]: (it's slightly more readable and will work if later the internal implementation of InstancesSelection is changed to use something else than an array).

**Comment by blurymind** [comment]
Thank you for taking the time to review :)

**Comment by 4ian** [comment]
Looks good to me! 🚀
Thanks for this nice addition!

**Pull request merged**